be/src/exec/schema_scanner/schema_partitions_scanner.cpp

```diff
@@ -22,10 +22,14 @@
 #include <stdint.h>
 
 #include "exec/schema_scanner/schema_helper.h"
-#include "runtime/decimalv2_value.h"
-#include "runtime/define_primitive_type.h"
-#include "util/runtime_profile.h"
+#include "runtime/client_cache.h"
+#include "runtime/exec_env.h"
+#include "runtime/runtime_state.h"
+#include "schema_scanner_helper.h"
```

Comment:
```suggestion
#include "exec/schema_scanner/schema_scanner_helper.h"
```
and notice the include order